shared/src/business/entities/cases/ElectronicPetition.ts

```diff
@@ -178,11 +174,6 @@ export class ElectronicPetition extends JoiValidationEntity {
         '*': 'Your Petition file size is empty',
         'number.max': `Your Petition file size is too big. The maximum file size is ${MAX_FILE_SIZE_MB}MB.`,
       }),
-    petitionRedactionAcknowledgement: joi.boolean().when('petitionType', {
```

Comment:
Is there any harm in leaving this in since it'll be added back with 10339? Not a whole lot of point in validating something we don't save here, but also not a whole lot of point in removing it